examples/v1beta1/fpga/OWNERS

```diff
@@ -0,0 +1,3 @@
+reviewers:
+  - eliaskoromilas # inaccel
+  - jstamel        # inaccel
```

Comment:
Let's create approvers list, so you can approve the PRs under this folder.

```suggestion
approvers:
  - eliaskoromilas
  - jstamel
```

Reply:
sure